edit-post/components/sidebar/settings-header/index.js

```diff
@@ -24,13 +24,19 @@ const SettingsHeader = ( { count, openDocumentSettings, openBlockSettings, sideb
 				onClick={ openDocumentSettings }
 				className={ `edit-post-sidebar__panel-tab ${ sidebarName === 'edit-post/document' ? 'is-active' : '' }` }
 				aria-label={ __( 'Document settings' ) }
+				data-label={ __( 'Document' ) }
 			>
 				{ __( 'Document' ) }
 			</button>
 			<button
 				onClick={ openBlockSettings }
 				className={ `edit-post-sidebar__panel-tab ${ sidebarName === 'edit-post/block' ? 'is-active' : '' }` }
 				aria-label={ __( 'Block settings' ) }
+				data-label={ (
+					1 === count ?
```

Comment:
This is a bit of a gnarly expression to put in a prop… plus it'd be better if we stored this in a variable and then just output it here and below. I'll make that tweak and push the change though. 😄